examples/orbit_5k_pipeline.ipynb

```diff
@@ -4,18 +4,29 @@
    "cell_type": "markdown",
```

Comment:
<p>1)\mathbb R is not properly rendered</p><p>2) add reference to persformer paper</p><p>3) &nbsp;<em>It maps a persistence diagram to the logarithm of the class probability.</em> That's not quite true, it's mapping it to the logits.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/giotto-ai/giotto-deep/pull/99/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>